cobigen/cobigen-core-systemtest/src/test/java/com/devonfw/cobigen/systemtest/TemplateProcessingTest.java

```diff
@@ -73,35 +73,36 @@ public void cleanup() {
   }
 
   /**
+   * Tests if template sets can be extracted properly
+   *
    * @throws IOException if an Exception occurs
    */
-  @Test
   public void extractTemplateSetsTest() throws IOException {
```

Comment:
this method should be private if it isn't a test.

Reply:
Adjusted.